src/PrestaShopBundle/Controller/Admin/Sell/Catalog/AttributeGroupController.php

```diff
@@ -95,12 +123,39 @@ public function createAction()
      *
      * @param int $attributeGroupId
      *
-     * @return RedirectResponse
+     * @return Response
      */
-    public function editAction(int $attributeGroupId)
+    public function editAction(Request $request, int $attributeGroupId): Response
     {
-        //@todo: implement in antoher pr
-        return $this->redirectToRoute('admin_attribute_groups_index');
+        $editableAttributeGroup = $this->getQueryBus()->handle(new GetAttributeGroupForEditing($attributeGroupId));
+        $attributeGroupFormBuilder = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Builder\AttributeGroupFormBuilder');
+        $attributeFormHandler = $this->get('PrestaShop\PrestaShop\Core\Form\IdentifiableObject\Handler\AttributeGroupFormHandler');
+
+        $attributeGroupForm = $attributeGroupFormBuilder->getFormFor($attributeGroupId);
```

Comment:
Once you have the form, and it's filled wi the data you can get the name from it with something like:
```suggestion
        $attributeGroupForm = $attributeGroupFormBuilder->getFormFor($attributeGroupId);
        $attributeGroupName = $attributeGroupForm->getData()['name'][$this->getContextLangId()];
```

Reply:
Using `getContextLangId` should return the language of the logged-in employee, instead of the global one common to all employees, thus the interface is better adapted for the actual user.

Reply:
Thats nice idea, didn't think about using existing form. Added it.